capture.cpp

```diff
@@ -1657,7 +1703,10 @@ const char *locale = DEFAULT_LOCALE;
     printf(" Darkframe: %s\n", yesNo(darkframe));
     printf(" Debug Level: %d\n", debugLevel);
     printf(" TTY: %s\n", yesNo(tty));
-    printf("%s\n", KNRM);
+    printf(" Version 0.8 Exposure Method: %s\n", yesNo(use_new_exposure_algorithm));

```

Comment:
Could you switch that to "Continuous Capture Method" and invert `use_new_exposure_algorithm, to be more clear about what this method is/does.